src/main/java/core/basesyntax/service/impl/StringParserImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.StringParser;
+import core.basesyntax.strategy.FruitTransaction;
+import core.basesyntax.strategy.FruitTransactionStrategy;
+import java.util.List;
+
+public class StringParserImpl implements StringParser {
+    private static final int INDEX_TRANSACTION = 0;

```

Comment:
```suggestion
    private static final int TRANSACTION_INDEX = 0;

```
all for other names. It's just grammaticaly correct)